### ISSUE TITLE
Push to talk (PTT) in 1:1 calls

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-web#15284]
https://github.com/vector-im/element-web/issues/5993 tracks this for jitsi conferences, we should support it for 1:1 calls too.

**Comment by dbkr** [comment]
For completeness, https://github.com/vector-im/element-web/issues/1358 track voice messages, which is a different thing.

**Comment by BloodyIron** [comment]
So does that mean PTT is finally coming to Element???

**Comment by BloodyIron** [comment]
Considering this request is a duplicate of #5993 can we get this closed and the discussion moved into #5993?

**Comment by SimonBrandner** [comment]
> #5993 tracks this for jitsi conferences, we should support it for 1:1 calls too.

@BloodyIron ...

**Comment by BloodyIron** [comment]
Yes it is, #5993 calls for the same functionality and was initiated about 2 years earlier. PTT be it 1:1 or rooms.

**Comment by SimonBrandner** [comment]
The functionality would be implemented in a very different way for Jitsi and for native Matrix calls, so it is _not_ a duplicate

**Comment by grand-lotus-iroh** [comment]
I wish there was Push To Talk... insane how this is common sense enough to have baked in by default. So many potato chip dorito eating neckbeards living in moms basement crushing ice cubes in people's ear.

**Comment by Thatoo** [comment]
Any update on this feature?

**Comment by grand-lotus-iroh** [comment]
> Any update on this feature?

Dead in the water just like the project itself... 3.3k issues.

**Comment by BloodyIron** [comment]
> > Any update on this feature?
> 
> Dead in the water just like the project itself... 3.3k issues.

You think Element is dead as a project? : https://github.com/element-hq/element-web/commits/develop/
